config/sync/taxonomy.vocabulary.pm_audience.yml

```diff
@@ -0,0 +1,8 @@
+uuid: 99fe0691-95b4-4828-af8f-b7a4309882b6
+langcode: en
+status: true
+dependencies: {  }
+name: 'PM Audience'
+vid: pm_audience
```

Comment:
Change to `audience`.  This isn't that specific to performance measures. Will require you to change the file name and every reference to this name.